moabb/evaluations/evaluations.py

```diff
@@ -5,6 +5,7 @@
 
 import numpy as np
 from mne.epochs import BaseEpochs
+from mne.utils import ProgressBar
```

Comment:
```suggestion
from tqdm import tqdm
```
It is better to depend directly on tqdm